src/devtools/client/debugger/src/actions/breakpoints/modify.js

```diff
@@ -78,15 +82,28 @@ export function enableBreakpoint(cx, initialBreakpoint) {
   };
 }
 
+async function trackBreakpoint(workspace) {
+  let context = {};
+  if (workspace?.name) {
+    context = { workspaceName: workspace.name };
+  }
+  mixpanel.track("breakpoint", context);
+}
+
 export function addBreakpoint(
   cx,
   initialLocation,
   options = {},
   disabled = false,
+  shouldTrack = false,
```

Comment:
this makes me nervous that somewhere there is a callsite that expects `shouldCancel`  to be the 5th param...

Reply:
Good to be nervous, I just checked again and all the callsites seem good.